File: node/lib/common_wrappers.js
require('shelljs/global');


function printError($msg){
    exec(__dirname + "/../../common/print.sh -error " + "'" + $msg + "'").output;
}

function printSuccess($msg){
  exec(__dirname + "/../../common/print.sh -success "  + "'" + $msg + "'").output;
}

module.exports = {
  printSuccess: printSuccess,
  printError: printError
}

File: ruby/lib/common_wrappers.rb
def print_error(msg)
    system(File.dirname(__FILE__) + "/../../common/print.sh -error " + "'" + msg + "'")
end

def print_success(msg)
  system(File.dirname(__FILE__) + "/../../common/print.sh -success "  + "'" + msg + "'")
end
